index.html

```diff
@@ -33,28 +30,45 @@ <h2>Telemetry dashboards</h2>
                     </a>
                     <a class="list-group-item" href="new-pipeline/evo.html">
                         Evolution Dashboard <div class="dashboard-description">View evolution of measure values over time.</div>
                     </a>
+                    <a class="list-group-item" href="https://metrics.mozilla.com/firefox-hardware-report/">
+                        Firefox Hardware Report
+                        <div class="dashboard-description">See what hardware Firefox users have.</div>
+                    </a>
                     <a class="list-group-item" href="https://sql.telemetry.mozilla.org/">
-                        SQL Portal <div class="dashboard-description">Query telemetry data with SQL.</div>
+                        SQL Portal <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">Query telemetry data with SQL.</div>
+                    </a>
+                    <a class="list-group-item" href="https://moz-experiments-viewer.herokuapp.com/">
+                        Experiments viewer <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">See the impact of different experiments have on our metrics.</div>
                     </a>
                     <a class="list-group-item" href="https://analysis.telemetry.mozilla.org/">
-                        Analysis Portal <div class="dashboard-description">Analyze telemetry data with Spark.</div>
+                        Analysis Portal <span class="fa fa-lock"></span>
+                        <div class="dashboard-description">Analyze telemetry data with Spark.</div>
                     </a>
-                    <a class="list-group-item" href="https://reports.telemetry.mozilla.org/">
-                        Reports Portal <div class="dashboard-description">Explore data analyses &amp; reports.</div>
+                    <h3>Documentation</h3>
+                    <a class="list-group-item" href="https://docs.telemetry.mozilla.org/">
+                        Data docs <div class="dashboard-description">Find what data is available &amp; how to use it.</div>
+                    </a>
+                    <a class="list-group-item" href="https://firefox-source-docs.mozilla.org/toolkit/components/telemetry/telemetry/">
+                        Firefox Telemetry docs <div class="dashboard-description">Learn how to record data &amp; what data gets sent.</div>
+                    </a>
+                    <a class="list-group-item" href="https://georgf.github.io/fx-data-explorer/">
+                        Probe dictionary <div class="dashboard-description">Find what probes are submitted from Firefox.</div>
                     </a>
                 </div>
             </div>
             <div class="col-md-4">
-                <h3>Telemetry @ Twitter</h3>
+                <h3>What is new</h3>
```

Comment:
bikeshed alert: Maybe prefer "What's new" as it is more commonly written with the contraction.